docs/requirements.txt

```diff
@@ -0,0 +1,190 @@
+google-pasta==0.2.0
+google-resumable-media==0.5.1
+googleapis-common-protos==1.52.0
+grpcio==1.31.0
+h5py==2.10.0
+heapdict==1.0.1
+idna==2.10
+imagesize==1.2.0
+importlib-metadata==1.7.0; python_version < "3.8"
+ipykernel==5.3.4
+ipython==7.18.1
+ipython-genutils==0.2.0
+ipywidgets==7.5.1
+isort==4.3.21
+itsdangerous==1.1.0
+jedi==0.17.2; python_version >= "3.3"
+jinja2==2.11.2
+joblib==0.16.0
+jsonschema==3.2.0
+jupyter==1.0.0
+jupyter-client==6.1.7
+jupyter-console==6.2.0
+jupyter-core==4.6.3
+keras-preprocessing==1.1.2
+kfp==1.0.0
+kfp-server-api==1.0.0
+kiwisolver==1.2.0
+kornia==0.1.4.post2
+kubernetes==11.0.0
+locket==0.2.0
+markdown==3.2.2
+markupsafe==1.1.1
+matplotlib==3.3.1
+mccabe==0.6.1
+mistune==0.8.4
+more-itertools==8.5.0
+msgpack==1.0.0
+nbconvert==5.6.1
+nbformat==5.0.7
+notebook==6.1.4
+numpy==1.17.5
+nuscenes-devkit==1.0.2
+oauthlib==3.1.0
+opencv-python==4.4.0.42
+opt-einsum==3.3.0
+packaging==20.4
+pandas==1.1.2
+pandocfilters==1.4.2
+parso==0.7.1; python_version >= "3.3"
+partd==1.1.0
+pathspec==0.8.0
+pbr==5.5.0
+pexpect==4.8.0; python_version >= "3.3" and sys_platform != "win32"
+pickleshare==0.7.5; python_version >= "3.3"
+pillow==7.2.0
+plotly==4.9.0
+pluggy==0.13.1
+prometheus-client==0.8.0
+prompt-toolkit==3.0.7
+protobuf==3.13.0
+psutil==5.7.2
+ptyprocess==0.6.0; python_version >= "3.3" and sys_platform != "win32" or os_name != "nt"
+py==1.9.0
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
+pycocotools==2.0.2
+pycodestyle==2.6.0
+pycparser==2.20
+pyflakes==2.2.0
+pygments==2.6.1
+pyparsing==2.4.7
+pyquaternion==0.9.5
+pyrsistent==0.17.0
+pytest==5.4.3
+pytest-cov==2.10.1
+python-dateutil==2.8.1
+pytorch-ignite==0.3.0
+pytz==2020.1
+pywin32==228; sys_platform == "win32"
+pywinpty==0.5.7; os_name == "nt"
+pyyaml==5.3.1
+pyzmq==19.0.2
+qtconsole==4.7.7
+qtpy==1.9.0
+recommonmark==0.6.0
+regex==2020.7.14
+requests==2.24.0
+requests-oauthlib==1.3.0
+requests-toolbelt==0.9.1
+responses==0.10.16
+retrying==1.3.3
+rsa==4.6; python_version >= "3.5"
+scikit-learn==0.23.2
+scipy==1.4.1
+send2trash==1.5.0
+shapely==1.7.1
+six==1.15.0
+snowballstemmer==2.0.0
+sortedcontainers==2.2.2
+sphinx==3.2.1
+sphinx-click==2.5.0
+sphinx-rtd-theme==0.5.0
+sphinxcontrib-applehelp==1.0.2
+sphinxcontrib-devhelp==1.0.2
+sphinxcontrib-htmlhelp==1.0.3
+sphinxcontrib-jsmath==1.0.1
+sphinxcontrib-qthelp==1.0.3
+sphinxcontrib-serializinghtml==1.1.4
+strip-hints==0.1.9
+tabulate==0.8.7
+tblib==1.7.0
+tensorboard==2.3.0
+tensorboard-plugin-wit==1.7.0
+tensorboardx==2.1
+tensorflow==2.3.0
+tensorflow-estimator==2.3.0
+termcolor==1.1.0
+terminado==0.8.3
+testpath==0.4.4
+threadpoolctl==2.1.0
+toml==0.10.1
+toolz==0.10.0
+torch==1.4.0
+torchvision==0.5.0
+tornado==6.0.4
+tqdm==4.48.2
+traitlets==5.0.4
+typed-ast==1.4.1
+typing-extensions==3.7.4.3
+urllib3==1.25.10
+wcwidth==0.2.5
+webencodings==0.5.1
+websocket-client==0.57.0
+werkzeug==1.0.1
+wheel==0.35.1
+widgetsnbextension==3.5.1
+wrapt==1.12.1
+yacs==0.1.8
+zict==2.0.0
+zipp==3.1.0; python_version < "3.8"
```

Comment:
Are those dependencies just for readthedocs?

Reply:
This file contain dependencies for the whole repo. Unfortunately I can't export only doc dependencies to requirements.ext file for poetory.